### PR TITLE
feat: denort binary

### PR DESCRIPTION
**Pull request by lucacasonato**
This adds the `denort` lite binary for standalone releases. Stripped linux size: 20mb

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -184,13 +185,15 @@ jobs:
...
           zip -r deno-x86_64-apple-darwin.zip deno
+          zip -r denort-x86_64-apple-darwin.zip denort
 
       - name: Pre-release (windows)
         if: |
           startsWith(matrix.os, 'windows') &&
           matrix.kind == 'test_release'
         run: |
           Compress-Archive -CompressionLevel Optimal -Force -Path target/release/deno.exe -DestinationPath target/release/deno-x86_64-pc-windows-msvc.zip
+          Compress-Archive -CompressionLevel Optimal -Force -Path target/release/denort.exe -DestinationPath target/release/denort-x86_64-pc-windows-msvc.zip
```
Comment (ry) [inline .github/workflows/ci.yml:196]: Just to be sure, these zip files won't be on the release page?

File: cli/main_runtime.rs [hunk 2]
```diff
@@ -0,0 +1,32 @@
+// Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
+
+#![deny(warnings)]
```
Comment (ry) [inline cli/main_runtime.rs:3]: Remove?

File: cli/standalone.rs [hunk 3]
```diff
@@ -1,45 +1,111 @@
...
+  s: S,
+) -> Result<S::Ok, S::Error>
+where
+  S: Serializer,
+{
+  match maybe_level {
+    None => s.serialize_none(),
+    Some(level) => s.serialize_str(&level.to_string()),
+  }
+}
```
Comment (ry) [inline cli/standalone.rs:106]: This seems like a lot of code to serialize the log level...

File: cli/tests/integration_tests.rs [hunk 4]
```diff
@@ -4994,6 +4994,21 @@ fn standalone_runtime_flags() {
...
+    .current_dir(util::root_path())
+    .stderr(std::process::Stdio::piped())
+    .spawn()
+    .unwrap()
+    .wait_with_output()
+    .unwrap();
+  assert!(!output.status.success());
+  let stderr_str = String::from_utf8(output.stderr).unwrap();
+  assert!(util::strip_ansi_codes(&stderr_str)
+    .contains("This executable is used internally by 'deno compile', it is not meant to be invoked directly."));
```
Comment (ry) [inline cli/tests/integration_tests.rs:5009]: Maybe checking the exact error message isn't necessary. Having a check that it returns a non-zero error code is good tho.

**Review by ry — APPROVED** [review]
LGTM

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bartlomieju) [reply .github/workflows/ci.yml:196]: Correct

File: cli/main_runtime.rs (hunk 2 above)
Comment (bartlomieju) [reply cli/main_runtime.rs:3]: Nope, don't want warnings slipping through CI

File: cli/standalone.rs (hunk 3 above)
Comment (bartlomieju) [reply cli/standalone.rs:106]: Unfortunately... I already expressed it in https://github.com/denoland/deno/pull/8738#discussion_r551052751

Comment (bartlomieju) [reply cli/standalone.rs:106]: Actually, just needed to activate "serde" feature in "log" crate

**Pull request merged**